ppddm-agent/src/main/scala/ppddm/agent/controller/prepare/DataPreparationController.scala

```diff
@@ -182,6 +182,7 @@ object DataPreparationController {
             case e: Exception =>
               val msg = s"An unexpected error occurred while preparing data"
               logger.error(msg)
+              logger.error(e.getMessage, e)
```

Comment:
According to our logging convention, it will be better to write "logger.error(msg, e)"

Reply:
Updated accordingly.